shared/devices/remove-device/index.js

```diff
@@ -29,19 +29,9 @@ RemoveDevice.propTypes = {
 
 export default connect(
   null,
-  dispatch => {
+  (dispatch, ownProps) => {
```

Comment:
cleaning this up everywhere. turns out we get ownprops in all these calls so we can skip the thing where we essentially do this in the 3rd parameters where we're merging. a lot cleaner this way